MANUAL.txt

```diff
@@ -954,7 +955,7 @@ Options affecting specific writers {.options}
 
 :   Use the [`listings`] package for LaTeX code blocks. The package
     does not support multi-byte encoding for source code. To handle UTF-8
-    you would need to use a custom template. This issue is fully
+    you would need to use a custom template. This issue is fully 
```

Comment:
Please avoid spaces at the end of the line, as in this line and some of the others.